manifests/0000_90_cloud-credential-operator_01_prometheusrole.yaml

```diff
@@ -4,6 +4,12 @@ metadata:
   name: prometheus-k8s
   namespace: openshift-cloud-credential-operator
 rules:
+- apiGroups:
+  - ""
+  resources:
+  - namespace/metrics
```

Comment:
Is this some special kube resource?

Reply:
Not sure. I was using the cluster-machine-approver repo as an example (b/c they also use controller-runtime) and they had it https://github.com/openshift/cluster-machine-approver/blob/master/manifests/0000_90_cluster-machine-approver_01_prometheusrole.yaml#L7-L12

Reply:
I don't see this in other repos, so I've removed it from the PR.

https://github.com/openshift/cluster-ingress-operator/blob/master/manifests/0000_90_ingress-operator_00_prometheusrole.yaml
https://github.com/openshift/cluster-dns-operator/blob/master/manifests/0000_90_dns-operator_00_prometheusrole.yaml

Reply:
So I tested this by removing this from the role, but you need to wait for the token to expire before the role changes take effect. It turns out that this is needed as the kube-rbac-proxy pod will show errors otherwise:

```
I0617 23:24:14.610086       1 proxy.go:102] Forbidden (user=system:serviceaccount:openshift-monitoring:prometheus-k8s, verb=get, resource=namespace, subresource=metrics)
```

What isn't clear to me is why other components do not need this permission, and I'll have to look into this.

Reply:
After discussion with the monitoring team, they asked for a BZ to investigate why this permissions is needed, but it seems like we're okay to go with the current role https://bugzilla.redhat.com/show_bug.cgi?id=1848600